tests/system/small/test_remote_function.py

```diff
@@ -882,6 +897,23 @@ def add_ints(row):
         pd_result, bf_result, check_dtype=False, check_exact=False
     )
 
+    # Read back the deployed BQ remote function using read_gbq_function.
+    func_ref = session.read_gbq_function(
+        function_name=add_ints_remote.bigframes_remote_function,  # type: ignore
+        is_row_processor=True,
+    )
+
+    assert add_ints_remote.bigframes_remote_function  # type: ignore
+    assert add_ints_remote.bigframes_cloud_function  # type: ignore
+    assert func_ref.bigframes_remote_function
```

Comment:
redundant, as we already have

assert add_ints_remote.bigframes_remote_function
assert func_ref.bigframes_remote_function == add_ints_remote.bigframes_remote_function

Reply:
Removed.